beacon_node/http_api/src/lib.rs

```diff
@@ -4448,6 +4448,23 @@ pub fn serve<T: BeaconChainTypes>(
             })
         });
 
+    // GET lighthouse/supply/{state_root}
+    let get_lighthouse_supply = warp::path("lighthouse")
+        .and(warp::path("supply"))
```

Comment:
```suggestion
    // GET lighthouse/analysis/global_validator_supply/{state_root}
    let get_lighthouse_supply = warp::path("lighthouse")
        .and(warp::path("analysis"))
        .and(warp::path("global_validator_supply"))
```

I think this fits the criteria for an `analysis` endpoint.
Also I think it might be worth using a more explicit name but I can't find an option I like. `global_validator_supply` seems a bit verbose, but `global_balances` seems a bit vague.

Reply:
I've changed the endpoint's path.